yugabyte_cloudformation.yaml

```diff
@@ -0,0 +1,490 @@
+          CidrIp: !FindInMap
+            - SubnetConfig
+            - VPC
+            - CIDR
+        - IpProtocol: tcp
+          FromPort: 9042
+          ToPort: 9042
+          CidrIp: !FindInMap
+            - SubnetConfig
+            - VPC
+            - CIDR
+      Tags:
+        - Key: 'Name'
+          Value: 'YugaByteSecurityGroup'
+
+  Node0NetInt:
+      Type: 'AWS::EC2::NetworkInterface'
+      Properties:
+        SubnetId: !Ref PublicSubnet0
+        GroupSet:
+          - !Ref YugaByteNodeSG
+  
+  Node1NetInt:
+      Type: 'AWS::EC2::NetworkInterface'
+      Properties:
+        SubnetId: !Ref PublicSubnet1
+        GroupSet:
+          - !Ref YugaByteNodeSG
+  
+  Node2NetInt:
+      Type: 'AWS::EC2::NetworkInterface'
+      Properties:
+        SubnetId: !Ref PublicSubnet2
+        GroupSet:
+          - !Ref YugaByteNodeSG
+
+
+  YugaByteNode0:
+    Type: 'AWS::EC2::Instance'
+    DependsOn: PublicRoute
+    CreationPolicy:
+      ResourceSignal:
+        Count: '1'
+        Timeout: PT5M
+    Metadata:
+       'AWS::CloudFormation::Init':
+        configSets:
+          InstallAndRun:
+            - Install
+            - Configure
+        Install:
+          packages:
+            yum:
+              git: []
+        Configure:
+          files:
+            /home/ec2-user/install_software.sh:
+              source: "https://raw.githubusercontent.com/vp15591/yugabytedb-cloudformation/master/scripts/install_software.sh"
+              mode: '0755'
+              owner: ec2-user
+              group: ec2-user        
+            /home/ec2-user/create_universe.sh:
+              source: "https://raw.githubusercontent.com/vp15591/yugabytedb-cloudformation/master/scripts/create_universe.sh"
+              mode: '0755'
+              owner: ec2-user
+              group: ec2-user
+            /home/ec2-user/start_master.sh:
+              source: "https://raw.githubusercontent.com/vp15591/yugabytedb-cloudformation/master/scripts/start_master.sh"
+              mode: '0755'
+              owner: ec2-user
+              group: ec2-user
+            /home/ec2-user/start_tserver.sh:
+              source: "https://raw.githubusercontent.com/vp15591/yugabytedb-cloudformation/master/scripts/start_tserver.sh"
+              mode: '0755'
+              owner: ec2-user
+              group: ec2-user    
+          commands:
+            01_Install_YugaByte_DB:
+              command:  bash /home/ec2-user/install_software.sh !FindInMap [ YugaByteConfig, Edition, Value ] !Ref DBVersion !FindInMap [ YugaByteConfig, Download, URL ]
+            02_Create_universe:
+              command: !Sub |
+                bash /home/ec2-user/create_universe.sh AWS ${AWS::Region} 3 ${Node0NetInt.PrimaryPrivateIpAddress} ${Node1NetInt.PrimaryPrivateIpAddress} ${Node2NetInt.PrimaryPrivateIpAddress} ${PublicSubnet0.AvailabilityZone} ${PublicSubnet1.AvailabilityZone} ${PublicSubnet2.AvailabilityZone} ${PublicSubnet0.AvailabilityZone}
```

Comment:
make the rf come from the variable instead of hard coded value

Reply:
Yes, I will make it come from the variable.